sufia-models/app/services/sufia/generic_file_indexing_service.rb

```diff
@@ -6,6 +6,9 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('file_format')] = object.file_format
         solr_doc[Solrizer.solr_name('file_format', :facetable)] = object.file_format
         solr_doc['all_text_timv'] = object.full_text.content
+
+        int_store = Solrizer::Descriptor.new(:integer, :indexed, :stored);
```

Comment:
Why are we indexing this?  Why not just `(:integer, :stored)`? Also, how about making this a constant?